tests/integration/targets/test_zabbix_mediatype/tasks/main.yml

```diff
@@ -150,6 +150,115 @@
     - assert:
         that: zbxmediatype_concur_fail.failed is sameas True
 
+  - when: zabbix_version is version('5.0', '>=')
+    block:
+    - name: test - reset email mediatype smtp information to default
+      zabbix_mediatype:
+      register: zbxmediatype_reset
+
+    - assert:
+        that: zbxmediatype_reset.changed is sameas True
+
+    - name: test - update email mediatype with message templates
+      zabbix_mediatype:
+        message_templates:
```

Comment:
Can you please add `eventsource: internal` ?

Reply:
done